### PR TITLE
Add design issue template

### LINKED ISSUE #1667: Adding a design issue template
## Problem
For contributors wanting to propose design changes, there is no clear way to share their ideas from the issues template list. The existing templates cover discussion and implementation issues but leave behind the ideation and iteration stage related to design.

## Description
Since we decided to separate design from dev issues to record design history and steer design discussions through [separated tickets](https://github.com/WordPress/openverse-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%96%BC%EF%B8%8F+aspect%3A+design%22), it would be ideal having a design issue template in the list.

The content structure I have applied goes as follows:

```
## Problem
Describe the design problem.

### Issues related
List all the issues related to the problem.

## Proposal
Share your design proposal through mockups, prototypes, or videos.

## Description
Describe the changes and new elements the design proposal has. Please mention all the components and styles impacted by your idea.

## Mockups
Add the Figma file name and link.
```

## Additional context
Our contribution guidelines also separate ways of [contributing through code and design](https://github.com/WordPress/openverse/blob/main/CONTRIBUTING.md#-design-contributions). This template will come to strengthen that track.

## Implementation
- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by shreyanshmalvya**
## Fixes

Fixes #1667 by @panchovm 

## Description

Added an issue template for design-related changes at : ```.github\ISSUE_TEMPLATE\design_changes.md```

## Testing Instructions

Reports no error upon executing ``` pnpm dev```

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by shreyanshmalvya** [comment]
Should I add the label ```✨ goal: improvement``` as well to the template as they might be related to design improvements?

File: .github/ISSUE_TEMPLATE/design_changes.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Design Changes
+about: Propose design Changes
```
Comment (sarayourfriend) [inline .github/ISSUE_TEMPLATE/design_changes.md:3]:
```suggestion
about: Propose design changes

```

**Review by sarayourfriend — CHANGES_REQUESTED** [review]
Just one small typo fix but otherwise LGTM :)

**Comment by sarayourfriend** [comment]
@shreyanshmalvya adding the improvement label makes sense to me. I say go for it :+1:

**Comment by shreyanshmalvya** [comment]
I have fixed the typo and added the ```✨ goal: improvement``` label

**Review by sarayourfriend — APPROVED** [review]
LGTM. Thanks :rocket:

**Comment by shreyanshmalvya** [comment]
@obulat could you please review this PR once.

File: .github/ISSUE_TEMPLATE/design_changes.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+name: Design Changes
+about: Propose design changes
+labels: "🖼️ aspect: design, ✨ goal: improvement, 🚦 status: awaiting triage"
+title: "<Replace this with actual title>"
+---
+
+## Problem
+<!-- Describe the design problem. -->
+
+### Issues related
```
Comment (dhruvkb) [inline .github/ISSUE_TEMPLATE/design_changes.md:11]: This bit confused me a little.

```suggestion
### Related issues

```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Looks good to me, except this small textual change.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**